client/src/main/proto/spine/event_annotations.proto

```diff
@@ -38,25 +38,19 @@ import "google/protobuf/descriptor.proto";
 
 extend google.protobuf.MessageOptions {
     // An instruction to associate an enrichment message with the event message to enrich.
+    // If the event message to enrich is in another package, the event type name must be fully-qualified.
```

Comment:
It would be better to say that the type name should be qualified according to Protobuf conventions. It may not be a full package name, just the path that it needed to resolve the type name.